.github/workflows/run-terraform.yml

```diff
@@ -72,11 +72,6 @@ on:
         description: "Required when using vault in terraform. Enables fetching jwt and logging in to vault for the terraform provider to work"
         required: false
         type: string
-      image_url:
```

Comment:
Denne er referert til ved deklarering av env variabler to steder under.. disse må vel også fikses?
Søk etter "inputs.image_url"...

Reply:
bra spotta :)